mp/src/game/client/viewpostprocess.cpp

```diff
@@ -786,6 +787,12 @@ static float GetCurrentBloomScale( void )
 	{
 		flCurrentBloomScale = mat_bloomscale.GetFloat();
 	}
+#ifdef NEO
+	if (!g_pMaterialSystemHardwareConfig->GetHDREnabled())
+	{
+		flCurrentBloomScale *= 4; // NEOTOD (Adam) the base values defined in neo_bloom_controller in old maps seem way too small, need to figure out what the right multiplier is for this. 4 seems to be in the right ballpark. The translation from non-hdr to hdr is 0.3 -> 1.0 according to valve developer community, a multiplier of 3.333 could be the right value
```

Comment:
We should not be going off of "old maps". We're remaking the game, maps "included" (recompile on sdk2013). So we shouldn't be setting this arbitrarily. Lets have a discussion before settling on this.

Reply:
If this multiplier is based off of how the bloom in the original looked, then I don't see an issue with this. Maybe some comparison shots would be good to have.
If things don't look right after we recompile maps with the new tools, then we can change the bloom value in the entity for those maps specifically, which then won't affect the intended look of custom maps that haven't been compiled for the new engine

Reply:
We planned on recompiling all maps for RE. If that isn't what we're doing then yeah this is fine.

Reply:
I'm talking about custom maps for OG specifically - it's down to the original creators of them to recompile those, otherwise we should make an attempt to have the bloom values as close to original as possible for them, since we can simply modify the values for the official maps when we go to recompile them, and can't for those
But it's not really a big deal. The lighting looks pretty different as-is so the changes in bloom probably aren't even noticeable at first glance

Reply:
Yeah good point. Well, whenever Zwia comes back I'll just leave this here for him to say his piece and resolve this.